subsys/logging/Kconfig

```diff
@@ -37,6 +37,21 @@ config LOG_BACKEND_RPC_BUFFER_SIZE
 	  Defines the maximum log line length that can be sent to the log forwarder
 	  over nRF RPC without fragmentation.
 
+config LOG_BACKEND_RPC_CRASH_LOG
+	bool "nRF RPC crash log"
+	select RETENTION
+	select RETENTION_MUTEX_FORCE_DISABLE
```

Comment:
Are these mutexes used anywhere?

Reply:
Not anymore after I force-disable them. :) They're used internally by the retention API, which is undesired given that we use the API from the fatal error handler.